src/content/ui/adaptive-responsive/large-screens.md

```diff
@@ -0,0 +1,237 @@
+[`SliverGridDelegateWith<b>MaxCrossAxisExtent</b>`]:  {{site.api}}/flutter/rendering/SliverGridDelegateWithMaxCrossAxisExtent-class.html
+
+:::
+Don't use the grid delegate for these classes that lets
+you set the column count directly and then hardcode
+the number of columns based on whether the device
+is a tablet, or whatever. 
+The number of columns should be based on the size of
+the window and not the size of the physical device.
+
+This distinction is important because many mobile
+devices support multi-window mode, which can
+cause your app to be rendered in a space smaller than
+the physical size of the display. Also, Flutter apps
+can run on web and desktop, which can be sized in
+many ways. <strong>For this
+reason, use `MediaQuery` to get the app window size
+rather than the physical device size.</strong>
+:::
+
+You can solve this scenario in one of two ways:
+* Wrap the `GridView`in a `ConstrainedBox` and give
+  it a `BoxConstraints` with a maximum width set.
+* Use a `Container` instead of a `ConstrainedBox`
+  if you want other functionality like setting the
+  background color.
+
+For choosing the maximum width value,
+consider using the values recommended
+by Material 3 in the [Applying layout][] guide.
+
+[Applying layout]: https://m3.material.io/foundations/layout/applying-layout/window-size-classes
+
+## Foldables
+
+As mentioned previously, Android and Flutter both
+recommend in their design guidance **not**
+to lock screen orientation,
+but some developers lock screen orientation anyway.
+Be aware that this can cause problems when running your
+app on a foldable device.
+
+When running on a foldable, the app might look ok
+when the device is folded. But when unfolding,
+you might find the app letterboxed.
+
+As described in the [SafeArea & MediaQuery][] page,
+letterboxing means that the app's window is locked to
+the center of the screen while the window is
+surrounded with black.
+
+Why can this happen?
+
+This can happen when using `MediaQuery` to figure out
+the window size for yur app. When the device is folded,
+orientation is restricted to portrait mode.
+Under the hood, `setPreferredOrientations` causes
+Android to use a portrait compatibility mode and the app
+is displayed in a letterboxed state.
+In the letterboxed state, `MediaQuery` never receives
+the larger window size that allows the UI to expand.
+
+You can solve this in one of two ways:
+
+* Support all orientations.
+* Use the dimensions of the _physical display_.
+  In fact, this is one of the _few_ situations where
+  you would use the physical display dimensions and
+  _not_ the window dimensions.
+
+How to obtain the physical screen dimensions?
+
+You can use the [`Display`][] API, introduced in
+Flutter 3.22/Dart 3.4, which contains the size,
+pixel ratio, and the refresh rate of the physical device. 
+
+<b>PENDING: Your notes said Flutter 3.13 introduced Display, but isn't it actually Flutter 3.22?</b>
```

Comment:
we added Display API in Flutter 3.13 https://medium.com/flutter/whats-new-in-flutter-3-13-479d9b11df4d#:~:text=Improved%20foldable%20support

Reply:
Thanks for clarifying!